Random FunctionalJavascript test failures on GitHub Actions

## Problem/Motivation
Our FunctionalJavascript tests seem to have random failures when run in our GitHub actions CI workflow.  Especially `AZPublicationInlineContributorTest::testInlineContributor()`.

### Describe the bug

Example failure message:
```
PHPUnit 9.6.23 by Sebastian Bergmann and contributors.

Runtime:       PHP 8.3.23
Configuration: /home/runner/work/az_quickstart/az_quickstart/az-quickstart-scaffolding/web/profiles/custom/az_quickstart/phpunit.xml.dist

F                                                                   1 / 1 (100%)

Time: 04:34.202, Memory: 14.00 MB

There was 1 failure:

1) Drupal\Tests\az_publication\FunctionalJavascript\AZPublicationInlineContributorTest::testInlineContributor
TypeError: i is not a constructor
    at t.widget (http://172.17.0.1:8080/core/assets/vendor/jquery.ui/ui/widget-min.js?v=10.4.8:9:897)
    at http://172.17.0.1:8080/core/misc/dialog/dialog.jquery-ui.js?v=10.4.8:7:5
    at http://172.17.0.1:8080/core/misc/dialog/dialog.jquery-ui.js?v=10.4.8:75:3
RangeError: Maximum call stack size exceeded
    at new t.<computed>.<computed> (http://172.17.0.1:8080/core/assets/vendor/jquery.ui/ui/widget-min.js?v=10.4.8:1:1)
    at new t.<computed>.<computed> (http://172.17.0.1:8080/core/assets/vendor/jquery.ui/ui/widget-min.js?v=10.4.8:1:1)
    at new t.<computed>.<computed> (http://172.17.0.1:8080/core/assets/vendor/jquery.ui/ui/widget-min.js?v=10.4.8:1:1)
    at new t.<computed>.<computed> (http://172.17.0.1:8080/core/assets/vendor/jquery.ui/ui/widget-min.js?v=10.4.8:1:1)
    at new t.<computed>.<computed> (http://172.17.0.1:8080/core/assets/vendor/jquery.ui/ui/widget-min.js?v=10.4.8:1:1)
    at new t.<computed>.<computed> (http://172.17.0.1:8080/core/assets/vendor/jquery.ui/ui/widget-min.js?v=10.4.8:1:1)
    at new t.<computed>.<computed> (http://172.17.0.1:8080/core/assets/vendor/jquery.ui/ui/widget-min.js?v=10.4.8:1:1)
    at new t.<computed>.<computed> (http://172.17.0.1:8080/core/assets/vendor/jquery.ui/ui/widget-min.js?v=10.4.8:1:1)
    at new t.<computed>.<computed> (http://172.17.0.1:8080/core/assets/vendor/jquery.ui/ui/widget-min.js?v=10.4.8:1:1)
    at new t.<computed>.<computed> (http://172.17.0.1:8080/core/assets/vendor/jquery.ui/ui/widget-min.js?v=10.4.8:1:1)

/home/runner/work/az_quickstart/az_quickstart/az-quickstart-scaffolding/web/core/tests/Drupal/FunctionalJavascriptTests/WebDriverTestBase.php:138
/home/runner/work/az_quickstart/az_quickstart/az-quickstart-scaffolding/vendor/phpunit/phpunit/src/Framework/TestResult.php:729

FAILURES!
Tests: 1, Assertions: 8, Failures: 1.
Error: Process completed with exit code 1.
```

## Proposed resolution
Look into how to prevent random failures.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -238,7 +238,7 @@ jobs:
           SYMFONY_DEPRECATIONS_HELPER: weak
           SIMPLETEST_DB: mysql://root:@127.0.0.1:${{ job.services.mysql.ports[3306] }}/db
           SIMPLETEST_BASE_URL: http://172.17.0.1:8080
-          MINK_DRIVER_ARGS_WEBDRIVER: '["chrome", {"browserName":"chrome","chromeOptions":{"args":["--disable-gpu","--headless", "--no-sandbox"]}}, "http://127.0.0.1:${{ job.services.chromedriver.ports[9515] }}"]'
+          MINK_DRIVER_ARGS_WEBDRIVER: '["chrome", {"browserName":"chrome","chromeOptions":{"args":["--disable-gpu","--headless","--disable-dev-shm-usage","--no-sandbox"]}}, "http://127.0.0.1:${{ job.services.chromedriver.ports[9515] }}"]'
 
   install:
     name: drush site:install

